test/cpp_extensions/libtorch_agnostic_extension/test/test_libtorch_agnostic.py

```diff
@@ -52,6 +52,23 @@ def _run_identity(prior_mem):
             curr_mem = torch.cuda.memory_allocated(device)
             self.assertEqual(curr_mem, init_mem)
 
+    def test_my_ones_like(self, device):
```

Comment:
This test is not run in CI but is added for ease of local development + in preparation for the future when we can run unit tests for cpp extensions.